packages/loot-core/src/types/handlers.d.ts

```diff
@@ -13,5 +14,6 @@ export interface Handlers
     BudgetHandlers,
     FiltersHandlers,
     NotesHandlers,
+    RulesHandlers,
```

Comment:
Not specifically needed here, but I noticed it was missing when looking through the untyped files.